### ISSUE TITLE
Web3 Goals

### ISSUE THREAD

**Issue by kiv1n** [livepeer/Grant-Program#132]
**Give a 3 sentence description for your proposal.**

Web3 Goals is an space where a person can get extra social and financial motivation to achieve their own goals.

It's not just an app. It is a space that is being created to help all of us become better developers, artists, entrepreneurs, athletes, people.

We hope the following features help fulfill the mission:

- Soulbound tokens that will motivate you to be more responsible about own goals.
- Set a goal and stake funds on it, that will only come back if you succeed.
- Sharing the stake between the people who started watching the goal and trying to motivate the author.
- Goals that can be validated on-chain or off-chain.
- Group marathons where people want to achieve the same goal.
- Journal to record steps towards the goal.
- AI that helps set goals and achieve them.
- Private goals with friends.
- And a lot more...

**Describe the problem you are solving.** 

People always set ambitious goals for themselves, but most often do not achieve them.

It not only frustrates and demotivates them, but it also has a detrimental effect to everyone. Because a person who has became better has an impact on the world around him. And also makes it better.

So the problem of motivating and supporting people is very important to all of us.

**Describe the solution you are proposing and how it will have a positive impact on the Livepeer developer ecosystem.** 

The core of the application is social actions. For example, proofs of goal achievement and motivational messages. 

I think if these elements are in the form of a video, people will be more engaged and motivated.

If this hypothesis works, Web3 Goals and Livepeer will have many positive effects in the form of new users and interactions.

**Describe why you are the right team with the capability to build this.**

I already have experience in developing social blockchain applications. For example, I participated in the creation of a decentralized reputation and justice system for Web3 (https://github.com/yourJustice-Live/).

I also participated in guerrilla marketing at saas, where my team and I attracted over 15,000 users.

**Describe the scope of the project including a 3 month timeline and milestones.** 

1. Feb 2023. Release version 1.0 with staking, watchers motivators, notifications, account page with history and attract users using guerilla marketing. KPI is 50 users with goals, 100 watchers.

2. Mar 2023. Release version 2.0 with validated goals, watchers super motivators, motivation and proof videos. KPI is 100 users with goals, 200 watchers.

3. Apr 2023. Release version 2.5, group marathons, journal. KPI is 200 users with goals, 400 watchers.

**Please estimate hours spent on project based on the above and how much funding you will need.** 

I think I've spent over 350 hours, including a few idea pivots. 

This is the current version of the application - https://web3goals.space/
This is project GitHub - https://github.com/web3goals

In the second version of the app, I want to integrate Livepeer to allow people to post and share videos with proofs and motivational messages. $1,900 will allow me to do that.

**Comment by hansy** [comment]
Thanks for applying @kiv1n. We'll review your application and get back to you shortly!

**Comment by kiv1n** [comment]
Thank you! There are some updates to the project. I built a prototype at the ETHforAll hackathon - https://devfolio.co/projects/web-goals-fbd8

In this app, I integrated Livepeer so people could post videos with proof of how they achieved their goals. And my current plan is to release the first version of the app with Livepper in mainnet in March 2023.

With your microgrant, I will be able to release the app with all planned features on time and offer a better user experience.
